src/components/date/__internal__/navbar/navbar.component.tsx

```diff
@@ -1,23 +1,27 @@
 import React from "react";
-import PropTypes from "prop-types";
 import StyledButton from "./button.style";
 import StyledNavbar from "./navbar.style";
 import Icon from "../../../icon";
 
-const Navbar = ({ onPreviousClick, onNextClick, ...props }) => (
+export interface NavbarProps {
+  onPreviousClick?: () => void;
+  onNextClick?: () => void;
+  className?: string;
```

Comment:
**question**: do we need a `className` here? This is an internal component only, and while I see we have tests for the `className` prop working, we don't actually pass this anywhere. (Maybe I missed something.) If this isn't needed, lets just remove this from the interface (I notice it wasn't there before) and remove all the spreading props onto `StyledNavbar`.

Reply:
We need to support className as `react-day-picker` passes a class down to the navbar to handle the layout, I could probably remove the type and leave it as a spread but I'd prefer to type it and then explicitly set the className so it's more readable for the next person working on this

![image](https://github.com/Sage/carbon/assets/44157880/16a26b69-1316-4bc5-a172-7346ef871280)